modules/lidar_oscillation.py

```diff
@@ -0,0 +1,75 @@
+"""
+Representing a LiDAR Oscillation
+"""
+
+
+class LidarReading:
```

Comment:
we already have a data structure for lidar detections so use that instead. it's called LidarDetection